main.yml

```diff
@@ -406,6 +406,8 @@ edge:
         title: Fleet Management
         group: fleet-management
       - id: manage-images
+        permissions:
+          method: isHidden
```

Comment:
The `isHidden` is not permission but just a normal flag under nav items.

```yml
- id: manage-images
  isHidden: true
  title: Manage images
```
(if the validator fails add the attribute to the list in the validate-yaml.js file)